token/program-2022-test/tests/confidential_transfer.rs

```diff
@@ -121,6 +121,46 @@ impl ConfidentialTokenAccountMeta {
         }
     }
 
+    async fn new_with_memo<T>(token: &Token<T>, owner: &Keypair) -> Self
```

Comment:
nit: I thought this created something with a memo, so how about calling this `new_with_required_memo_transfers`?

Reply:
Yep, `new_with_required_memo_transfers` is much better. Thanks!